### DISCUSSION TITLE
Possible to normalize one sensor to another when using a Mix sensor?

### DISCUSSION THREAD

**Discussion by crawfish3141** [Rem0o/FanControl.Releases#577] [Ideas]
My CPU is an AMD 5800x, GPU is an RTX 3060 Ti, and I have front and rear case fans. The 5800x throttles once it reaches 90C, and so I have my case fans running full blast a little lower than that at 85C, using only the CPU as input. They will cool the GPU down during CPU-intensive tasks that don't tax the GPU. For example, the CPU/GPU idle temps might be 60/50 with inaudible fans, while the load temps are 90/40 with the fans at full blast. I'm struggling to use the Mix sensor functions to keep the CPU response as it is while still ramping up the case fans when the GPU temp increases, and the CPU doesn't increase enough to really engage my fan curves.

I think I could achieve what I want if the Max function allowed me to normalize the GPU temps to the CPU, e.g. multiplying the GPU temps by (say) 1.2 for the purpose of the sensor. Is this possible, or could it be made possible? :)

**Comment by Rem0o** [comment]
Try the mix curve instead. Simply create one curve per sensor and you get exactly what you want.

**Comment by crawfish3141** [comment]
I'm not seeing how to do it. Mix Curves have me add fan curves, and their output is a fan speed percentage. If I apply that to, say, my Rear fan control, it makes that fan's speed whatever the Mix Curve percentage is. That would seem to make my current configuration impossible, because it has CPU fan minimum at 40% and case fan at 30%.

**Reply by Rem0o** [reply]
Create 3 fan curves for your case fans. 
1: Graph with input as CPU temp
2: Graph with input as GPU temp
3: Mix fan curve set to max with input curve 1 and curve 2 above

Use 3 on your rear fan control.

**Reply by crawfish3141** [reply]
That's what I did.

**Reply by Rem0o** [reply]
I don't understand the issue. 

With this setup => 
if CPU is hot => cpu graph increases => mix will use that %
if GPU gets hot => gpu graph increases => mix will use that %

Isn't that what you want?

**Comment by crawfish3141** [comment]
Yes, but my CPU minimum fan speed is 40%, while my case fan minimum speeds are 25 and 30%. The Mix Curve would raise the case fan minimums to 40%, which makes them audible.

**Reply by Rem0o** [reply]
Why are we talking about your CPU fan?

The curve I mentionned above are NEW curves for your CASE FANS, not for your cpu fan.
Both graphs should represent the % you want for your CASE FANS, based on each respective temperature for CPU and GPU.

**Reply by crawfish3141** [reply]
Got it. Very elegant. Thank you!

**Comment by Jonathanese** [comment]
The graph values are normalized by their nature. Since you have an input temperature, but an output 0-100%, and then mix graphs use the output % rather than the temperature values.

But if it helps in clarity, you could do a linear graph that is 0 at your minimum temperature, and then 100% at your max temperature. Then you can use your mix this way.
